test/integration/options.spec.js

```diff
@@ -415,4 +415,41 @@ describe('options', function () {
       }, path.join(__dirname, 'fixtures', 'options', 'help'));
     });
   });
+
+  describe('--exclude', function () {
+    /*
+     * Runs mocha in ./fixtures/options/exclude with the given args.
+     * Calls handleResult with the result
+     */
+    function runExcludeTest (args, handleResult, done) {
+      directInvoke(args, function (error, result) {
```

Comment:
unless there's a Good Reason to use `directInvoke()`, please use `runMocha()` instead